pkgs/applications/version-management/git-and-tools/git-vanity-hash/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "git-vanity-hash-unstable";
+  version = "2020-02-26";
```

Comment:
I'm not sure where to put the unstable. If it goes in the pname, then the pname doesn't match the attrpath which seems unfortunate. If you put it in the version, then the pname won't match the name returned by `builtins.parseDrvName`.

I think it is probably better to include "unstable" in the version, but I'm open to other suggestions. I'm sad I'm not aware of official consensus on this issue.

Reply:
Another nixpkgs committer agreed with me that it should be in the version.

Reply:
```suggestion
  version = "2020-02-26-unstable";
```

Reply:
Updated.